sdks/python/apache_beam/typehints/native_type_compatibility.py

```diff
@@ -163,8 +163,14 @@ def is_any(typ):
   return typ is typing.Any
 
 
+try:
+  _ForwardRef = typing.ForwardRef
+except AttributeError:
```

Comment:
Could you please add a comment which Python version requires the branching?

Reply:
done, also rebased